modules/openshift-architecture-common-terms.adoc

```diff
@@ -0,0 +1,186 @@
+// Module included in the following assemblies:
+//
+// * architecture/index.adoc
+
+:_content-type: REFERENCE
+[id="openshift-architecture-common-terms_{context}"]
+= Common terms
+
+This glossary defines common terms that are used in the architecture content. These terms help you understand {product-title} architecture effectively.
+
+access policies::
+A set of roles that dictate how users, applications, and entities within a cluster interacts with one another. Access policies increase cluster security.
+
+admission plug-ins::
+Admission plug-ins enforce security policies, resource limitations, or configuration requirements.
+
+authentication::
+To control access to an {product-title} cluster, a cluster administrator can configure user authentication and ensure only approved users access the cluster. To interact with an {product-title} cluster, you must authenticate to the {product-title} API. You can authenticate by providing an OAuth access token or an X.509 client certificate in your requests to the {product-title} API.
+
+bootstrap::
+A temporary machine that runs minimal Kubernetes and deploys the {product-title} control plane.
+
+certificate signing requests (CSRs)::
+A resource requests a denoted signer to sign a certificate. This request might get approved or denied.
+
+cluster Version Operator (CVO)::
```

Comment:
Cluster here should have stayed capitalized

Reply:
acked